src/operator/numpy/np_cumsum-inl.h

```diff
@@ -157,10 +157,10 @@ void CumsumBackwardImpl(const OpContext& ctx,
     }
   }
   Stream<xpu> *s = ctx.get_stream<xpu>();
-  MSHADOW_TYPE_SWITCH_WITH_BOOL(igrad.type_flag_, IType, {
+  MSHADOW_TYPE_SWITCH_WITH_BOOL(igrad.type_flag_, index_t, {
```

Comment:
Can you revert this change as well ? rest LGTM !

Reply:
Great point, reverted to IType.